InteractiveRunner.html

```diff
@@ -115,26 +115,26 @@
 }
 
 function startTest() {
-    var queryParam = parseQueryString['suite'];
+    const queryParam = parseQueryString['suite'];
     if (queryParam !== undefined)
         disableAllSuitesExcept(queryParam);
 
-    var runner = new BenchmarkRunner(Suites, {
-        willRunTest: function (suite, test) {
+    const runner = new BenchmarkRunner(Suites, {
+        willRunTest(suite, test) {
             test.anchor.classList.add('running');
         },
-        didRunTest: function (suite, test) {
-            var classList = test.anchor.classList;
+        didRunTest(suite, test) {
+            const classList = test.anchor.classList;
             classList.remove('running');
             classList.add('ran');
         },
-        didRunSuites: function (measuredValues) {
-            var results = '';
-            for (var suiteName in measuredValues.tests) {
-                var suiteResults = measuredValues.tests[suiteName];
-                for (var testName in suiteResults.tests) {
-                    var testResults = suiteResults.tests[testName];
-                    for (var subtestName in testResults.tests) {
+        didRunSuites(measuredValues) {
+            let results = '';
+            for (let suiteName in measuredValues.tests) {
+                let suiteResults = measuredValues.tests[suiteName];
+                for (let testName in suiteResults.tests) {
+                    let testResults = suiteResults.tests[testName];
+                    for (let subtestName in testResults.tests) {
```

Comment:
We can use const in these instances.

Reply:
Ah good point, I always forget about the for-in/for-of variant. 👍 

DONE